File: sqlite-android-studio-plugin/src/main/java/com.alecstrong.sqlite.android/lang/SqliteTokenSetQuoteHandler.java
package com.alecstrong.sqlite.android.lang;

import com.alecstrong.sqlite.android.SQLiteParser;
import com.intellij.codeInsight.editorActions.SimpleTokenSetQuoteHandler;

public class SqliteTokenSetQuoteHandler extends SimpleTokenSetQuoteHandler {
  public SqliteTokenSetQuoteHandler() {
    super(SqliteTokenTypes.TOKEN_ELEMENT_TYPES.get(SQLiteParser.QUOTE),
        SqliteTokenTypes.TOKEN_ELEMENT_TYPES.get(SQLiteParser.STRING_LITERAL));
  }
}

File: sqlite-android-studio-plugin/src/main/kotlin/com/alecstrong/sqlite/android/lang/SqliteTokenSetQuoteHandler.kt
package com.alecstrong.sqlite.android.lang

import com.alecstrong.sqlite.android.SQLiteParser
import com.intellij.codeInsight.editorActions.SimpleTokenSetQuoteHandler

class SqliteTokenSetQuoteHandler : SimpleTokenSetQuoteHandler(
    SqliteTokenTypes.TOKEN_ELEMENT_TYPES[SQLiteParser.QUOTE],
    SqliteTokenTypes.TOKEN_ELEMENT_TYPES[SQLiteParser.STRING_LITERAL])
